terraform/environment-pipelines/iam.tf

```diff
@@ -45,7 +45,7 @@ data "aws_iam_policy_document" "access_artifact_store" {
   statement {
     effect    = "Allow"
     actions   = ["codestar-connections:UseConnection", "codeconnections:ListTagsForResource"]
```

Comment:
Don't you need both `codeconnections:UseConnection` and `codestar-connections:UseConnection` to account for the old and new code* connections action sets?

Reply:
I think this is correct yes, I added it in the e2e tests but will need to add here too